Library/Source/BgfxCallback.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#ifndef WIN32
+#include <alloca.h>
+#define alloca(size)   __builtin_alloca(size)
+#endif
+
+#include <bgfx/bgfx.h>
+#include <bgfx/platform.h>
+#include <vector>
+
+namespace Babylon
+{
+    struct BgfxCallback : public bgfx::CallbackI
+    {
+        virtual ~BgfxCallback() = default;
+
+        virtual void fatal(const char* filePath, uint16_t line, bgfx::Fatal::Enum code, const char* str) override;
```

Comment:
Do the overrides need to be public? Usually I prefer making them private/protected.